apps/meteor/client/views/room/Sidepanel/SidepanelItem/useParentTeamData.ts

```diff
@@ -0,0 +1,63 @@
+import { TEAM_TYPE } from '@rocket.chat/core-typings';
+import type { SubscriptionWithRoom } from '@rocket.chat/ui-contexts';
+import { useEndpoint, useUserId } from '@rocket.chat/ui-contexts';
+import { useQuery, keepPreviousData } from '@tanstack/react-query';
+
+import { goToRoomById } from '../../../../lib/utils/goToRoomById';
+
+type APIErrorResult = { success: boolean; error: string };
```

Comment:
I think you can use `FailureResult` from `apps/meteor/app/api/server/definition.ts` instead of redeclaring it.